### ISSUE TITLE
Blender 3.0 support

### ISSUE THREAD

**Issue by elsandosgrande** [LuxCoreRender/BlendLuxCore#710]
The current latest build fails to open in 3.0 alpha, so I guess that my question is: “When might Blender 3.0 be supported?”

---

This is what I get when I try to enable the extension by the way:

![image](https://user-images.githubusercontent.com/40576902/125182519-f901db00-e20e-11eb-9ec4-939aabd07533.png)

There was a different error when I tried to enable it for the very first time — something mentioning three parameters when only one was expected or something — but I don't seem to be able to recreate it at the moment.

### Edit

So… Even though there is no check in the checkbox, it seems like LuxCore was still enabled. However…

![image](https://user-images.githubusercontent.com/40576902/125182781-2780b580-e211-11eb-97f6-b9dd510a9ee8.png)

**Comment by Theverat** [comment]
We can only support stable Blender versions. Under-development versions change from day to day and it's too much work too keep up with that.

**Comment by elsandosgrande** [comment]
All right. Thank you for your quick response.

**Comment by juangea** [comment]
@elsandosgrande it seems you have two installations of LuxCore, and the error you show tells you that, so be sure to remove any LuxCore folder you may have under addons and try adding it again.

Still it may not work since 3.0 is not supported because of continuous changes in the API, but it may work.

**Comment by elsandosgrande** [comment]
@juangea I actually removed the extension and installed it again right before taking that screenshot, since I wanted to get the initial error again. When I tried to check the checkbox for the first time, the error said something about some function expecting only one parameter and receiving three if I remember correctly, with subsequent attempts at checking the checkbox resulting in the error above.

I'll try it again after restarting the laptop with the extension removed from Blender.
